app/models/retention_schedule.rb

```diff
@@ -0,0 +1,21 @@
+class RetentionSchedule < ApplicationRecord
+
+  validates_presence_of(:case)
+  validates_presence_of(:erasure_date)
+  validates_presence_of(:status)
+
+  # this should change to Case::Base
+  # as this retention_schedule is expanded
+  # to other / all case types
+  belongs_to :case,
+             foreign_key: :case_id,
+             class_name: 'Case::SAR::Offender'
+
+  enum status: { 
+    review: "review", 
+    retain: "retain", 
+    erase: "erase", 
+    not_set: "not_set" 
+  }, _default: "not_set"
```

Comment:
@willmcb, a little bit confused by the name of the status,  does this status indicate the case has been reviewed, retained, erased, or it indicates the stage where the case is under?